_data/projects/reactive/releases/2.4/series.yml

```diff
@@ -5,7 +5,7 @@ maven:
       summary: Core implementation
 integration_constraints:
   java:
-    version: 11, 17, 20, 21, and 22
+    version: 11, 17, 20, 21, 22 or 23
```

Comment:
Should we be mentioning 20 ?

Reply:
I wouldn't have, but since we did mention it in the past, we can hardly backtrack now.